admin-dev/themes/new-theme/js/pages/product/components/combination-list/combinations-list-editor.ts

```diff
@@ -245,12 +251,20 @@ export default class CombinationsListEditor {
     }
   }
 
+  /**
+   * The product page doesn't incude the combination list into a for tag because it is rendered inside a form
```

Comment:
```suggestion
   * The product page doesn't include the combination list into a form tag because it is rendered inside a form
```

Reply:
The screenshots would bring nothing since there are no visual changes

Reply:
My bad I thought you were modifying / fixing some product form behavior.